src/forms/FormFieldAdvancedSelection/AdvancedSelectionDrawer.tsx

```diff
@@ -1,421 +1,45 @@
 import * as React from "react";
 import { ButtonProps } from "@root/components/Button";
-import { FieldDef } from "@root/components/Field";
 import {
-	ChangeEvent,
 	memo,
 	ReactElement,
 	useCallback,
 	useEffect,
-	useMemo,
-	useState,
-	useRef
+	useState
 } from "react";
-import Button from "../../components/Button";
-import Form, { formActions, useForm } from "@root/components/Form";
-import {
-	InputWrapper,
-	StyledInput,
-} from "./AdvancedSelection.styled";
-import AddIcon from "@mui/icons-material/Add";
-import ChipList from "./ChipList";
-import LoadMoreButton from "./LoadMoreButton";
+import { useForm } from "@root/components/Form";
 import { AdvanceSelectionDrawerPropTypes } from ".";
 import _ from "lodash";
-import { MosaicLabelValue } from "@root/types";
 import { FormDrawerWrapper } from "../shared/styledComponents";
-import { useRefs } from "../shared/refsContext/RefsContext";
+import { DataViewFilterMultiselectDropdownContent } from "@root/components/DataViewFilterMultiselect";
+import DrawerHeader from "@root/components/DrawerHeader";
 
 const AdvancedSelectionDrawer = (props: AdvanceSelectionDrawerPropTypes): ReactElement => {
 	const {
 		value,
 		fieldDef,
 		onChange,
-		isModalOpen,
-		isMobileView,
 		handleClose,
 		handleUnsavedChanges,
-		dialogOpen,

```

Comment:
We shouldn't be deleting the dialog functionality, this helps users know when they've made any changes to the component.